### ISSUE TITLE
Ctrl-F find results in "-1 of n matches"

### ISSUE THREAD

**Issue by jacobpennington821** [godotengine/godot#61713]
### Godot version

4.0.alpha9

### System information

Windows 10

### Issue description

Ctrl-F find can result in messages being displayed such as "-1 of 2 matches", when the correct behaviour should be "2 of 2 matches". This occurs when the search box is closed with text in it and then reopened without editing the contents of the search box.
![image](https://user-images.githubusercontent.com/22193839/172047347-d870df0e-a146-4e9d-a993-64c35160e636.png)

### Steps to reproduce

1. Open the following script in the godot editor 
```
extends Node


# Called when the node enters the scene tree for the first time.
func _ready():
	pass # Replace with function body.


# Called every frame. 'delta' is the elapsed time since the previous frame.
func _process(delta):
	pass
```
2. Ctrl-F and search for the `#` character
3. Hit Escape to close the search box
4. Ctrl-F again to reopen the search box.
5. Shift-Enter to go to the last match, "-1 of 3 matches" is displayed.

**Comment by KoBeWi** [comment]
Can't reproduce the -1, but the initial value is 0 when doing this (and when starting search with something selected), so there definitely is some bug here.
![godot_6dq4zRzb0V](https://user-images.githubusercontent.com/2223172/172048572-fcf3bd05-b8e8-4558-9a65-3c449f388af5.gif)

CC @rohanrhu

**Comment by rohanrhu** [comment]
Loooooking

**Comment by rohanrhu** [comment]
Hii, sent a PR that fixes this issue, seems like working correctly now.

**Comment by Paulb23** [comment]
Duplicate of #62949 fixed by #62952
